src/config.c

```diff
@@ -384,11 +385,11 @@ void parse_config (GromitData *data)
               exit (1);
             }
 
-          if (key_type == HOTKEY_SYMBOL_VALUE)
+          if (key_type == HOTKEY_SYMBOL_VALUE && g_strcmp0(data->hot_keyval, DEFAULT_HOTKEY) == 0)
             {
               data->hot_keyval = g_strdup(scanner->value.v_string);
             }
-          else
+          else if (key_type == UNDOKEY_SYMBOL_VALUE && g_strcmp0(data->hot_keyval, DEFAULT_UNDOKEY) == 0)
```

Comment:
C&P error here :-)

Reply:
That's embarrassing :sweat_smile:. Fixed in 7eed24f!